src/Xamarin.Android.Build.Tasks/Tests/Xamarin.ProjectTools/Xamarin.ProjectTools.csproj

```diff
@@ -9,72 +9,7 @@
   <Import Project="..\..\..\..\external\xamarin-android-tools\src\Microsoft.Android.Build.BaseTasks\MSBuildReferences.projitems" />
   <ItemGroup>
     <Compile Remove="Resources\**\*.cs" />
-    <EmbeddedResource Include="Resources\AndroidX\Tabbar.xml">
```

Comment:
I'm not sure it's "safe" to remove all of these, largely because of the `%(LogicalName)` overrides.

(That, and I can never remember the semantics for the default `%(LogicalName)` value…)

Reply:
The `%(LogicalName)` were all listing the default value over and over... that is why I removed them.